Synthetics default retention filter - docs update (#21375)
DataDog/documentation: content/en/synthetics/apm/_index.md

`x-datadog-sampling-priority: 1`
: To make sure that the Agent keeps the trace.

### How long are traces retained?

> These traces are retained for 15 days thanks to the `Synthetics Default` retention filter, [just like your classical APM traces][20].

Reviewer: ```suggestion
These traces are retained for 15 days with the `Synthetics Default` retention filter, [just like your classical APM traces][20].
```
